source/utilities.tex

```diff
@@ -4880,7 +4880,7 @@
 
 \pnum
 \ctype
-The type $\tcode{T}_I$.
+The $\tcode{I}^\text{th}$ element of \tcode{Types}, where indexing is zero-based.
```

Comment:
Why is this better than the original, given that you now have a widely usable definition of  T_i available? Maybe the original should also use code font for `I`.

Reply:
Probably due to what I said in https://github.com/cplusplus/draft/pull/4789#discussion_r681838323, but I was wrong there. [variant] also uses T<sub>_j_</sub> and T<sub>_I_</sub> (without code font `I`), so I suggest this:
```suggestion
$\tcode{T}_I$.
```

Reply:
But that's basically the status quo, isn't it?

Reply:
You're right.